bundles/org.openhab.transform.javascript/conf/transform/js/readme/readme.js

```diff
@@ -0,0 +1,4 @@
+(function(i) {
```

Comment:
If I understand correctly these are used in the tests. So, these should be moved to `src/test/resources`.

Reply:
I did it similarly than it's done in another file based transformation tests (map, scale and xslt). For consistency reason, I propose to merge this as it is and then change them all by another PR if necessary.

Reply:
Can you move them into `src/test/resources`? Would be awesome if you create another PR to fix the other transformations.

Reply:
Transformation service load transformation files from conf/transform/... folder so they can't be moved to `src/test/resources`, but I'm not sure why files should be copied(and removed) to the target during tests, because tests seems to work fine without copying.  I couldn't find  the person who have introduced the test to map, scale and xslt to ask the reason.

Reply:
Interesting... I think we're good to merge, then.